src/index.tsx

```diff
@@ -141,15 +186,22 @@ export default class AppIntroSlider<ItemT = any> extends React.Component<
     name: string,
     onPress?: (e: GestureResponderEvent) => void,
   ) => {
+    const leftButtonStyle = this.props.renderVertically
+      ? styles.topButtonContainer
+      : styles.leftButtonContainer
+    const rightButtonStyle = this.props.renderVertically
+      ? styles.bottomButtonContainer
+      : styles.rightButtonContainer
     const style =
       name === 'Skip' || name === 'Prev'
-        ? styles.leftButtonContainer
-        : styles.rightButtonContainer;
+        ? leftButtonStyle
+        : rightButtonStyle;
```

Comment:
I think we should stick to having a single class per right/left button and they we just render these vertically if that props is enabled.

Reply:
but for that we have to define style inside render function